Library/include/CSP/Systems/Users/Authentication.h

```diff
@@ -188,95 +153,16 @@ class CSP_API AgoraUserTokenParams
 {
 public:
 	csp::common::String AgoraUserId;
-	int Lifespan;
 	csp::common::String ChannelName;
+	int Lifespan;
 	bool ReadOnly;
 	bool ShareAudio;
 	bool ShareVideo;
 	bool ShareScreen;
 };
 
 
-/// @ingroup User System
-/// @brief @brief Data class used to contain information requesting a user token
-class CSP_API AgoraUserTokenResult : public csp::systems::ResultBase
```

Comment:
This and `CheckoutSessionUrlResult` were made private and moved just like `LogoutResult`.